oem/firstvoices/ios/build_keyboards.sh

```diff
@@ -102,3 +109,27 @@ if [ $DO_BUILD = true ] || [ $DO_COPY = true ]; then
 
   popd
 fi
+
+if [ $DO_DOWNLOAD = true ]; then
+  echo "Downloading keyboards from downloads.keyman.com"
+
+  cp ../keyboards.csv "$TARGET/keyboards.csv"
+
+  SCRIPT_ROOT=`pwd`
+  URL_DOWNLOAD=https://downloads.keyman.com
+  URL_API_VERSION=${URL_DOWNLOAD}/api/keyboard/
```

Comment:
So from the API Use Notice
> This internal-use-only API is intended for use by api.keyman.com and (*.)keyman.com and by maintenance tasks. It may change without notice. Do not use this API within Keyman or other apps or on 3rd party web sites.

Building keyboards is the maintenance task. And then we don't want apps internally using the API. 
Is that correct?

Reply:
That's correct. In this case, the build is controlled by us and not distributed to other users, so it is okay to depend on an internal API. That's no different than TeamCity dependencies for the build.